tests/extendrtests/src/rust/Cargo.toml

```diff
@@ -16,7 +16,13 @@ crate-type = ["staticlib"]
 extendr-api = "*"
 
 [patch.crates-io]
-extendr-api = { path = "../../../../../../../../extendr-api" }
-extendr-engine = { path = "../../../../../../../../extendr-engine" }
-extendr-macros = { path = "../../../../../../../../extendr-macros" }
-#libR-sys = { git = "https://github.com/extendr/libR-sys" }
+
+# development verion
+#extendr-api = { path = "../../../../extendr-api/"}
+#extendr-engine = { path = "../../../../extendr-engine/"}
+#extendr-macros = { path = "../../../../extendr-macros/"}
+
+# R CMD check
+extendr-api = { git = "https://github.com/extendr/extendr", branch = "metadata"}
+extendr-engine = { git = "https://github.com/extendr/extendr", branch = "metadata" }
+extendr-macros = { git = "https://github.com/extendr/extendr", branch = "metadata" }
```

Comment:
Could you use these relative paths:
https://github.com/extendr/extendr/blob/10d0ab8a64ab4fafb125d69cf555f08b5b090593/tests/extendrtests/src/rust/Cargo.toml#L18-L21

You can then check locally as described here (and GH Actions will work also):
https://github.com/extendr/extendr/blob/master/tests/extendrtests/README.md#running-tests-locally